src/translations.js

```diff
@@ -92,20 +94,75 @@ module.exports = class Translation {
   spawnTranslations () {
     let self = this;
     let alpha = moment();
-
-    console.info(`Spawning Translations started for ${self.config._LOCALES}`);
-
-    Promise.map(self.config._LOCALES, (translationLocale) => {
-      return self.downloadTranslationData(translationLocale).then( (transations) => {
-        self.updateCache(translationLocale, transations);
-        console.info(`Spawning Translations finished for ${translationLocale}`);
+    if(process.env.ORGANISATION=="CH"){
```

Comment:
CH is country , organisation should be cembra, you have to check what parameter's dev ops people are setting in the env

Reply:
1. Ok. I will change the parameter name to country.
2. I have discussed with anil about this so we added the parameter organisation .I have informed yousuf about this. He will be informing devops people to set it in the env.

Reply:
No what i meant is that this library should be agnostic of the organisation or country, this just should loach CH config like other countries does. That is it. This is a common modules. The consumer checks which country it has to fetch.

Reply:
So how can it be done?